django/core/management/commands/squashmigrations.py

```diff
@@ -184,6 +186,15 @@ def handle(self, **options):
 
         # Write out the new migration file
         writer = MigrationWriter(new_migration, include_header)
+
+        if os.path.isfile(writer.path):
+            raise CommandError(
+                'There already is a migration named %s. It is not safe to '
+                'overwrite that migration if it has been applied anywhere. '
+                'Try specifying a different name for the new squashed '
+                'migration.' % new_migration.name
```

Comment:
Maybe
```suggestion
                'Migration f{squashed_name} already exits, use a different file name.'
```
We should move this check to the [`squashmigrations` command](https://github.com/django/django/blob/main/django/core/management/commands/squashmigrations.py).

Reply:
I believe this already is `django/core/management/commands/squashmigrations.py`; did you mean something else?

Reply:
True, sorry.